src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -39,4 +39,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public boolean isMutating() {
+        return true;
```

Comment:
This method should have a header comment. Only then users of the overridden method will know how it exactly behaves